Brisanet/ion
# Add wait warning icon

## Description

Adding wait-warning icon

## How to Test

[Check the chromatic deploy](https://62eab350a45bdb0a5818520e-vhwywjaskk.chromatic.com/?path=/story/ion-navigation-tabgroup--tabs-with-icons)

## Screenshots

![image](https://user-images.githubusercontent.com/90062413/216332669-b368dec8-fd98-4a89-888e-b0606a32dd4a.png)

## Compliance
